src/oracledb_datapump/files.py

```diff
@@ -878,6 +880,8 @@ def resolve_sequences_for_export(
                 sequenced_file = make_seq_dumpfile(file.name, seq_num)
                 resolved_files.add(sequenced_file)
 
+        logger.debug("Resolved dumpfiles for export: %s, ")
+
```

Comment:
This seems to be missing an actual variable

Reply:
Good catch. Linter/Pylance never catches that. See revision.